### PR TITLE
be a bit more picky about calling badgesPermissionsModel

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
r?

This fixes the issue where webmaker.org/admin/badges doesn't list any badges.

What's weird about this is, it seems, removing this line: https://github.com/mozilla/webmaker.org/blob/master/Gruntfile.js#L61

Changes how we need to access the file.

It has something to do with this: https://github.com/mozilla/webmaker.org/blob/master/lib/badges-permissions-model.js#L68-L78

Without showdown, it fails on module.exports and instead adds it to window, with showdown it has a module.exports and thus never adds it to window.

File: public/js/angular/controllers.js [hunk 1]
```diff
@@ -377,7 +377,9 @@ angular
       $scope.reverse = false;
       $scope.predicate = 'created';
       $scope.hasPermissions = function (badge) {
-        return window.badgesPermissionsModel({
+        var badgesPermissionsModel = module.exports;
```
Comment (Pomax) [inline public/js/angular/controllers.js]: it feels like this should be possible as an include based on what's loaded up top. @gvn do you have any wisdom here?

Comment (gvn) [inline public/js/angular/controllers.js]: I'm confused about what's being done here. Why not just return an object literal instead?

**Comment by ScottDowne** [comment]
This is the real fix: https://github.com/k88hudson/makeapi-angular/pull/7

**Pull request closed without merging**